paymentsheet/src/test/java/com/stripe/android/paymentsheet/repositories/CustomerRepositoryTest.kt

```diff
@@ -30,12 +32,14 @@ import java.security.InvalidParameterException
 internal class CustomerRepositoryTest {
     private val testDispatcher = UnconfinedTestDispatcher()
     private val stripeRepository = mock<StripeRepository>()
+    private val errorReporter = FakeErrorReporter()
```

Comment:
We probably want an `@Before` that clears the logged errors from this, otherwise we might have spill over between tests.

Reply:
Done. In practice, I think this wasn't happening but I don't fully understand why